.github/workflows/tox.yml

```diff
@@ -0,0 +1,49 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on pull request events, or push events to master branch
+  push:
+    branches:
+      - main
```

Comment:
Silly me, `main` was simply because I was learning this Actions thingy with my new test repos which default to `main`...
I think it makes sense to suppress CI while actively hacking on a feature branch, but that's just a matter of taste.